perun/view_diff/report.py

```diff
@@ -567,6 +588,33 @@ def generate_trace_stats(graph: Graph) -> dict[str, list[TraceStat]]:
     return trace_stats
 
 
+def _get_baseline_target_total() -> tuple[list[float], list[float]]:
+    """A helper function to obtain baseline target total values for all stats.
+
+    :return: collections of baseline and target total values
+    """
+    # TODO: Nasty, but currently the only reasonable way to access the maxima.
+    return [
+        float(stat.value[0])
+        for stat in Config().profile_stats["baseline"]
+        if stat.name.startswith("Overall")
+    ], [
+        float(stat.value[0])
+        for stat in Config().profile_stats["target"]
+        if stat.name.startswith("Overall")
+    ]
+
+
+def _make_percentage(value: float) -> float:
```

Comment:
`_to_percentage`?  `_as_percentage`? That would sound more clearer, `make` sounds very vague.